docs/user/filesystem/partition_schemes.rst

```diff
@@ -0,0 +1,128 @@
+.. _fsds_partition_schemes:
+
+Partition Schemes
+=================
+
+Partition Schemes can be defined using a JSON/typesafe configuration or by a common name. By default these schemes all
+utilize "leaf storage" with a one up sequence number. This allows for appending data when it arrives out of order.
+
+Common Date schemes are:
+
+* minute
+* hourly
+* daily
+* monthly
+* julian-minute
+* julian-hourly
+* julian-daily
```

Comment:
weekly is possibly you can do any date format but its not currently a "commonly used one"